Objective-C/CBLConsoleLogger.m

```diff
@@ -45,7 +45,8 @@ - (void) logWithLevel: (CBLLogLevel)level domain: (CBLLogDomain)domain message:
     
     NSString* levelName = CBLLog_GetLevelName(level);
     NSString* domainName = CBLLog_GetDomainName(domain);
-    NSLog(@"CouchbaseLite %@ %@: %@", domainName, levelName, message);
+    os_log_t log = os_log_create("CouchbaseLite", "OSDebug");
```

Comment:
What is "OSDebug"? Where does it appear in the log?

Reply:
`os_log_create(const char *subsystem, const char *category)` is a place holder for category. These will not be present in the logs as we know them, but only on debugging via XCode. The userAgent will have the same exact shape/info. So I just put a placeholder without any meaning.

I can see that Jim did `private CoreFoundation.OSLog _logger = new CoreFoundation.OSLog("CouchbaseLite", "dotnet");` so maybe I should change to "Platform" instead of "OSDebug" and "OSLogging" ? IMO having the platform name is not relevant as it should be already known what platform it is.

Reply:
changed to `com.couchbase.lite.ios` and the category now represents a domain (CBL)